### PR TITLE
Rebalancing of all Lore Artifacts

### PR DESCRIPTION
**Pull request by arithon**
All lore artifacts rebalanced according to [Mystery Codex of Norgannon](https://docs.google.com/spreadsheets/d/1xDjRHfG-ovKVcEQVfv02BsK3j0dgwS2rfH7KJ4GirA0/edit?usp=drive_web&ouid=106017264017540018811). Massive nerf to librams.

### CODE REVIEW

File: common/artifacts/908_wc_artifacts_relic.txt [hunk 1]
```diff
@@ -88,16 +63,45 @@ chalice_of_trollbanes = {
...
 
 	light_group_opinion = 5
 
 	picture = "GFX_inv_misc_trophy_argent"
 	slot = relic
 }
 
+# Rare
+dark_crystal = {
+	quality = 5
```
Comment (MemeWizard) [inline common/artifacts/908_wc_artifacts_relic.txt]: > Rare

> quality 5

File: common/artifacts/909_wc_artifacts_library.txt [hunk 2]
```diff
@@ -66,14 +44,37 @@ spellbook_of_drekthar = {
...
 	combat_rating = 10
 
-	monthly_character_prestige = 0.75
+	monthly_character_prestige = 0.8
 
 	picture = "GFX_spellbook_of_drekthar"
 	slot = library
+}
+book_of_medivh = {
+	quality = 5
```
Comment (MemeWizard) [inline common/artifacts/909_wc_artifacts_library.txt]: > Rare

quality 5

File: common/artifacts/999_wc_artifacts_other.txt [hunk 3]
```diff
@@ -28,6 +28,19 @@ eye_of_sargeras = {
...
+remnant_of_caer_darrow = {
+	quality = 3
+	flags = { misc }
+
+	monthly_character_piety = 0.6
+
+	text_effect_powerful_magical_artifact = 1
+
+	picture = "GFX_remnant_caer_darrow"
+	slot = relic
```
Comment (MemeWizard) [inline common/artifacts/999_wc_artifacts_other.txt]: > other

> relic slot

File: common/artifacts/908_wc_artifacts_relic.txt (hunk 1 above)
Comment (arithon) [reply common/artifacts/908_wc_artifacts_relic.txt]: right. thx. moved it but forgot to change number. done

File: common/artifacts/909_wc_artifacts_library.txt (hunk 2 above)
Comment (arithon) [reply common/artifacts/909_wc_artifacts_library.txt]: done.

File: common/artifacts/999_wc_artifacts_other.txt (hunk 3 above)
Comment (arithon) [reply common/artifacts/999_wc_artifacts_other.txt]: done.

**Pull request merged**